services/traction/api/endpoints/routes/credentials.py

```diff
@@ -24,7 +24,7 @@
 
 
 from api.endpoints.dependencies.db import get_db
-from api.endpoints.dependencies.tenant_security import get_from_context
+from api.endpoints.dependencies.tenant_context import get_from_context
```

Comment:
i don't understand why we move a function to a new file like this?

Reply:
because tenant_security imports modesl.base.py now, so it's a circular import if they are in the same file

Reply:
ah, ok. regardless that tenant_context.get_from_context doesn't really apply at the db level (HTTP exception saying not authenticated isn't appropriate - or rather misleading)